packages/odyssey-storybook/.storybook/preview-head.html

```diff
@@ -13,6 +13,10 @@
 />
 
 <style>
+  html {
+    font-size: 14px;
+  }
```

Comment:
Shouldn't this be represented as a percentage of the browser default? `87.5%` ?

Reply:
Ideally, we'll pull in `var(--FontSizeBase)`, [similar to what Sunny is doing here](https://okta.slack.com/archives/C7T2H3KNJ/p1644429285531139?thread_ts=1644427810.956199&cid=C7T2H3KNJ), so that it picks up the theme when values are updated.

If it's desirable, we can write the CSS such that it converts to a `%` for optimum scaling. Something like: `font-size: calc((var(--FontSizeBase) / 16px) * 100%);`

Reply:
The key difference in Sunny's example is that she is using a percentage based `font-size` which will scale off the user's font size preference yea? In our case we're forcing the user to interact with our storybook at 14px even if they want a larger text size. At least that is my understanding!

Reply:
Sunny's implementation is a bit limited because she doesn't currently have a base-size token to reference. 

AFAIK, scaling should still occur as intended with a `px` size on `html`. If it does cause issues, we can either set the token to `87.5%` or use the `calc()` above.

Reply:
see my comment below!